### PR TITLE
Use yarn

### PR DESCRIPTION
**Pull request by saralohr**
This PR replaces `npm` with `yarn` to solve some dependency issues, and I guess `yarn` is faster and better these days than `npm`, and most of our other JS apps use it.

I've updated automated scripts and docs to change `npm` -> `yarn`. But let me know if I missed anything or in general if you see any issues with this approach.

### CODE REVIEW

**Review by bradcypert — APPROVED** [review]

**Review by bradjteamsnap — APPROVED** [review]
🧶 👍

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -33,13 +33,13 @@ jobs:
...
-        run: npm run build
+      - name: Yarn build
+        run: yarn run build
 
-      - name: NPM Audit
-        run: npm audit --audit-level=high
+      - name: Yarn Audit
+        run: yarn npm audit --severity high
 
 #       - run: npm test -- Eventually.
```
Comment (ryanwi) [inline .github/workflows/main.yml:45]: 😆 . Anything for yarn to test?

**Review by ryanwi — APPROVED** [review]

**Review by l3aconator — APPROVED** [review]
LGTM

**Review by edwinsoftwaredev — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (saralohr) [reply .github/workflows/main.yml:45]: 🤷🏼‍♀️ I'll remove this line for now

**Pull request merged**